scripts/runTUFClient.ts

```diff
@@ -0,0 +1,61 @@
+import * as core from '@actions/core';
```

Comment:
Would be nice to remove this actions dependency so you could run this locally.

You can crash the process by throwing an error instead of calling `core.setFailed`, and could replace `core.getInput` with some command line arguments to this file. 

You could add `minimist` as a dev dep to get slightly better arg parsing: `cost argv = require('minimist')(process.argv.slice(2));`